src/frontend/src/components/common/Breadcrumbs/interfaces.ts

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export interface BreadcrumbProps {
+  firstItem?: string;
+  secondItem?: string;
+  thirdItem?: string;
```

Comment:
Should we simply define the props as an array of strings? This would allow us to scale more easily.